lasagne/layers/cuda_convnet.py

```diff
@@ -196,7 +356,86 @@ def get_output_for(self, input, **kwargs):
 
 
 class MaxPool2DCCLayer(CCLayer):
-    def __init__(self, incoming, pool_size, ignore_border=False, stride=None,
+    """
+    2D max-pooling layer
+
+    Performs 2D max-pooling over the two trailing axes of a 4D input tensor
+    (or over axis 1 and 2 if ``dimshuffle=False``, see notes). This is an
+    alternative implementation which uses the cuda-convnet wrappers from
+    pylearn2: ``pylearn2.sandbox.cuda_convnet.pool.MaxPool``.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    pool_size : integer or iterable
```

Comment:
No :) There is just very little consistency in how the various data types are named in docstrings. We should probably figure this out in a separate issue and decide on a 'standard'.